src/server/enterprise/cmds/cmds.go

```diff
@@ -192,6 +201,50 @@ func GetStateCmd() *cobra.Command {
 	return cmdutil.CreateAlias(getState, "enterprise get-state")
 }
 
+func SyncContextsCmd() *cobra.Command {
+	syncContexts := &cobra.Command{
+		Short: "Pull all available Pachyderm Cluster contexts into your pachctl config",
+		Long:  "Pull all available Pachyderm Cluster contexts into your pachctl config",
+		Run: cmdutil.Run(func(args []string) error {
+			cfg, err := config.Read(false, false)
+			if err != nil {
+				return err
+			}
+
+			c, err := client.NewOnUserMachine("user")
```

Comment:
This should get an enterprise server client with `client.NewEnterpriseClientOnUserMachine("user")`.